src/hooks/useFirestoreConnect.ts

```diff
@@ -1,8 +1,8 @@
-// eslint-disable-next-line no-restricted-imports
 import {
   isEmpty as _isEmpty,
   isLoaded as _isLoaded,
   ReduxFirestoreQuerySetting,
+  // eslint-disable-next-line no-restricted-imports
   useFirestoreConnect as _useFirestoreConnect,
   WhereOptions,
 } from "react-redux-firebase";
```

Comment:
[note] This is the one place where we actually want this direct import allowed, thus disabling the lint warning.